README.md

```diff
@@ -321,6 +322,11 @@ end
 remote :app do
   "false"
 end
+
+# filtering - only runs on app hosts with an option of primary set to true
+remote :app, primary: true do
+  "migrate-task"
```

Comment:
Is it obvious that `migrate-task` is a shell command and not something provided by bootleg?